examples/example-dockpanel-amd/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
I'm wary of adding the other examples since they aren't TS and they can easily get out of date.

Reply:
Having them non TS is actually a good thing IMO (and why I wen't out of the way to convert them - including IE support).
While they shouldn't break for any minor/point releases they can certainly be added to the suite of tests (to check they load without warning) - I will look into this today?

Reply:
Further Note:  Once lumino has UMD support, then these types of examples work well in things like gist ObservableHQ and any online JS editor (the include paths will need to point to jasdelivr or unpkg or some other CDN server).

Reply:
👍 to including these with tests.

Reply:
@blink1073 you would think that would have been easy - ended up rolling my own with puppeteer